docs/sdkx-react-native/delegates.mdx

```diff
@@ -0,0 +1,227 @@
+---
+sidebar_position: 100
+title: Helpshift Delegates
+description: "The Helpshift SDK provides delegate callbacks to help app developers track a user's activities within the help section."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Helpshift Delegates {#delegates}
+
+<Intro>
+
+The Helpshift SDK provides delegate callbacks to help app developers track a user's activities within the help section.
+
+</Intro>
+
+## Helpshift Delegate implementation {#delegate-implementation}
+
+- To use Helpshift Delegate please import `helpshiftEmitter`
+```
+import {
+  helpshiftEmitter,
+} from 'helpshift-plugin-sdkx-react-native';
+
+```
+
+```
+const handleCallbacks = () => {
+    helpshiftEmitter.addListener('onEventOccurred', (data) => {
+      if (data.eventName == 'receivedUnreadMessageCount') {
+        try {
+          let countData =
+            Platform.OS == 'ios' ? data.eventData : JSON.parse(data.eventData);
+          console.log(countData.count);
+        } catch (err) {
+          console.error('Parsing Error', err);
+        }
+      } else {
+        console.log(JSON.stringify(data));
+      }
```

Comment:
Specifically mention that this event, receivedUnreadMessageCount, is handled as an example.
Also write a statement above, outside code block that we need to register a listener with helpshiftEmitter for "onEventOccurred" and "onUserAuthenticationFailure"

Reply:
Updated